src/Microsoft.VisualStudio.Composition/Reflection/TypeRef.cs

```diff
@@ -238,7 +238,7 @@ internal Type ResolvedType
         public static TypeRef Get(Resolver resolver, AssemblyName assemblyName, int metadataToken, string fullName, TypeRefFlags typeFlags, int genericTypeParameterCount, ImmutableArray<TypeRef> genericTypeArguments, bool shallow, ImmutableArray<TypeRef> baseTypes, TypeRef? elementTypeRef)
         {
             Requires.NotNull(resolver, nameof(resolver));
-            return new TypeRef(resolver, assemblyName, null, metadataToken, fullName, typeFlags, genericTypeParameterCount, genericTypeArguments, shallow, baseTypes, elementTypeRef);
+            return new TypeRef(resolver, resolver.GetNormalizedAssemblyName(assemblyName), null, metadataToken, fullName, typeFlags, genericTypeParameterCount, genericTypeArguments, shallow, baseTypes, elementTypeRef);
```

Comment:
Build break would be fixed by this, I think:

```suggestion
            return new TypeRef(resolver, Resolver.GetNormalizedAssemblyName(assemblyName), null, metadataToken, fullName, typeFlags, genericTypeParameterCount, genericTypeArguments, shallow, baseTypes, elementTypeRef);
```

Reply:
Yeah, I had uncommitted changes in my local checkout.

I will add tests for the PR aswell